src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -885,6 +899,7 @@ struct ConstraintGamma2Compute : ConstraintGamma2, db::ComputeTag {
       const tnsr::I<DataVector, SpatialDim, Frame>& coords) noexcept {
     destructive_resize_components(gamma, get<0>(coords).size());
     get(*gamma) = exp(-0.0078125 * get(dot_product(coords, coords))) + 0.001;
+    // get(*gamma) = 1.0;
```

Comment:
oops, sorry, looks like there were two of these.

Reply:
Derp, I should've caught that too... Thanks!